entities/src/main/java/org/odk/collect/entities/LocalEntityUseCases.kt

```diff
@@ -70,9 +70,8 @@ object LocalEntityUseCases {
         entitySource: EntitySource,
         mediaFile: MediaFile
     ) {
-        val newListHash = "server:${mediaFile.hash}"
```

Comment:
We're locked into using the server hash now that we need it outside `LocalEntityUseCase#updateLocalEntitiesFromServer` now so I don't think we need to deal with a prefix any longer. We could of course start reusing a prefix again if we want.